sqlx-postgres/src/column.rs

```diff
@@ -10,11 +10,21 @@ pub struct PgColumn {
     pub(crate) name: UStr,
     pub(crate) type_info: PgTypeInfo,
     #[cfg_attr(feature = "offline", serde(skip))]
-    pub(crate) relation_id: Option<i32>,
+    pub(crate) relation_id: Option<crate::types::Oid>,
     #[cfg_attr(feature = "offline", serde(skip))]
     pub(crate) relation_attribute_no: Option<i16>,
 }
 
+impl PgColumn {
+    pub fn relation_id(&self) -> Option<u32> {
+        self.relation_id.map(|oid| oid.0)
+    }
+
+    pub fn relation_attribute_no(&self) -> Option<i16> {
```

Comment:
```suggestion
    /// Returns the 1-based index of this column in its parent table, if applicable.
    ///
    /// This will be `None` if the column is the result of an expression.
    ///
    /// Corresponds to column `attnum` of the `pg_catalog.pg_attribute` table:
    /// <https://www.postgresql.org/docs/current/catalog-pg-attribute.html>
    pub fn relation_attribute_no(&self) -> Option<i16> {
```